### PR TITLE
updated iteration of signing keys (previously a list, now a map).

### PR DESCRIPTION
**Pull request by aricart**
[update] jwt2 now signing keys is a map, not a list - internal check for what changed when account loads and update copies them out, so the extraction needed to be different. Note this is referencing a non-released JWT library.

/cc @nats-io/core

### CODE REVIEW

**Comment by kozlovic** [comment]
@aricart As you can see, there is a test that was using this as a list. So this test should be updated.

**Comment by aricart** [comment]
> @aricart As you can see, there is a test that was using this as a list. So this test should be updated.

yes! - didn't show in my usages for some reason. Fixed now.

File: server/accounts.go [hunk 1]
```diff
@@ -2844,11 +2844,10 @@ func (s *Server) updateAccountClaimsWithRefresh(a *Account, ac *jwt.AccountClaim
 	// update account signing keys
 	a.signingKeys = nil
 	signersChanged := false
-	if len(ac.SigningKeys) > 0 {
-		// insure copy the new keys and sort
-		a.signingKeys = append(a.signingKeys, ac.SigningKeys...)
-		sort.Strings(a.signingKeys)
+	for k, _ := range ac.SigningKeys {
```
Comment (kozlovic) [inline server/accounts.go:2847]: staticcheck flags this: replace with `for k := ..`

**Review by kozlovic — CHANGES_REQUESTED** [review]

**Review by matthiashanel — COMMENTED** [review]
LGTM - pending staticcheck change

**Review by kozlovic — APPROVED** [review]
LGTM

**Review by matthiashanel — APPROVED** [review]
LGTM

**Pull request merged**